client/js/options.js

```diff
@@ -19,6 +19,7 @@ const options = $.extend({
 	notifyAllMessages: false,
 	part: true,
 	quit: true,
+	seconds: false,
```

Comment:
Maybe change this to "showSeconds" because it makes sense here, but everywhere else (data.msg.seconds) it isn't clear exactly what it does.